Added Turkish translation

ernince opened the pull request:
Related to https://github.com/nt4f04uNd/sweyer/issues/10

nt4f04uNd commented on lib/localization/arb/intl_tr.arb:
```diff
@@ -0,0 +1,325 @@
+  },
+  "track": "Parça",
+  "tracks": "Parçalar",
+  "tracksPlural": "{count, plural, =1{{count} track} other{{count} tracks}}",
```
Could you also translate this one?
Here you need to translate the words "track" and "tracks"

nt4f04uNd commented on lib/localization/arb/intl_tr.arb:
```diff
@@ -0,0 +1,325 @@
+    "description": "Uyarı: Bu karakterlerin xml çıkışlı olması gerekir: Tüm tırnak işaretleri ve &<>"
+  },
+  "albums": "Albümler",
+  "albumsPlural": "{count, plural, =1{{count} album} other{{count} albums}}",
```
Same here

nt4f04uNd commented on lib/localization/arb/intl_tr.arb:
```diff
@@ -0,0 +1,325 @@
+    "description": "Uyarı: Bu karakterlerin xml çıkışlı olması gerekir: Tüm tırnak işaretleri ve &<>"
+  },
+  "playlists": "Oynatma listeleri",
+  "playlistsPlural": "{count, plural, =1{{count} playlist} other{{count} playlists}}",
```
Same here

nt4f04uNd commented on lib/localization/arb/intl_tr.arb:
```diff
@@ -0,0 +1,325 @@
+    "description": "Uyarı: Bu karakterlerin xml çıkışlı olması gerekir: Tüm tırnak işaretleri ve &<>"
+  },
+  "artists": "Sanatçılar",
+  "artistsPlural": "{count, plural, =1{{count} artist} other{{count} artists}}",
```
Same here

nt4f04uNd commented on lib/localization/arb/intl_tr.arb:
```diff
@@ -0,0 +1,325 @@
+    },
+    "description": Uyarı: Bu karakterlerin tekil sürümde xml çıkışlı olması gerekir: Tüm tırnak işaretleri ve &<>"
+  },
+  "deletePlaylists": "{count, plural, =1{Delete playlist} other{Delete {count} playlists}}",
```
Same here

nt4f04uNd commented on lib/localization/arb/intl_tr.arb:
```diff
@@ -0,0 +1,325 @@
+    "description": "Uyarı: Bu karakterlerin xml çıkışlı olması gerekir: Tüm tırnak işaretleri ve &<>"
+  },
+  "delete": "Sil",
+  "deleteTracks": "{count, plural, =1{Delete track} other{Delete {count} tracks}}",
```
Same here

nt4f04uNd commented on lib/localization/arb/intl_tr.arb:
```diff
@@ -0,0 +1,325 @@
+    "description": "Çalma listesi ekranındaki + düğmesinin başlığı"
+  },
+  "remove": "Kaldır",
+  "removeTracks": "{count, plural, =1{Remove track} other{Remove {count} tracks}}",
```
Same here

nt4f04uNd commented on lib/localization/arb/intl_tr.arb:
```diff
@@ -0,0 +1,325 @@
+      "count": {}
+    }
+  },
+  "deletePlaylistsConfirmation": "{count, plural, =1{Are you sure you want to delete {title}?} other{Are you sure you want to delete selected playlists?}}",
```
Same here

nt4f04uNd reviewed (commented):
Hi
First of all - thank you for contributing!

This is already good, but there are a few translation you perhaps missed. It would be ideal if the localization for a language was complete. Could you take a look at those?

ernince commented:
I updated it, you can report any other misses if available.

nt4f04uNd commented on lib/localization/arb/intl_tr.arb:
```diff
@@ -0,0 +1,325 @@
+        "example": "Parça adı"
+      }
+    },
+    "description": Uyarı: Bu karakterlerin tekil sürümde xml çıkışlı olması gerekir: Tüm tırnak işaretleri ve &<>"
```
There is a missing quote character here

ernince commented:
Done, also i checked the whole translation and changed misses. I guess it's complete now...

nt4f04uNd commented on lib/localization/arb/intl_tr.arb:
```diff
@@ -30,11 +30,11 @@
   },
   "album": "Albüm",
-  "albumQueue": "Albüm {name}",
+  "albumQueue": "Albüm: {name}",
```
Sorry to be picky, but please remove the `:` that you added here. It is absent in the original English string and is not needed there. Here and everywhere else wher you did this.

ernince replied:
It would be more understantable in Turkish. When it's translated it is a bit un-understandable, but as you want of course..

nt4f04uNd replied:
`{name}` is a variable. In the app we graphically separate the actual name from the `Album` part.
It looks like this

![_](https://user-images.githubusercontent.com/39104740/235369797-b08270fa-13a5-4d71-b68a-0dac923c7886.jpg)

ernince replied:
Ok, got it

nt4f04uNd commented on lib/localization/arb/intl_tr.arb:
```diff
@@ -30,11 +30,11 @@
     "placeholders": {
       "name": {
-        "type": "String",
+        "type": "Dizi",
```
You don't need to tnraslate these, please revert it

nt4f04uNd reviewed (commented):
And the last mile :-)

ernince commented:
I guess it's done now 😀

nt4f04uNd commented:
I fixed the strings format in the recent commit:
- `other` word in plurals was actually a keyword, so I fixed it back to be english
-  Changed the order of `{title}` variable in some plurals
- `removeTracksConfirmation` and `deleteTracksConfirmation` are supposed to mean different things, so I took the proper translation from `deletePlaylistsConfirmation`

Can you please take a look if you don't mind these changes and if they sound good in Turkish?

ernince commented:
> I fixed the strings format in the recent commit:
> - `other` word in plurals was actually a keyword, so I fixed it back to be english
> -  Changed the order of `{title}` variable in some plurals
> - `removeTracksConfirmation` and `deleteTracksConfirmation` are supposed to mean different things, so I took the proper translation from `deletePlaylistsConfirmation`
> 
> Can you please take a look if you don't mind these changes and if they sound good in Turkish?

 Of course. They are pretty fine, no problem.

nt4f04uNd commented:
Cool, thank you for your contribution!

I will also make sure to later update the https://github.com/nt4f04uNd/sweyer/issues/10 description with more instructions on how to translate `.arb` files, because it is quite complex for someone unfamiliar with this format.
